config/analytics.yml

```diff
@@ -154,6 +154,7 @@ shared:
     - predicted_grade
     - public_id
     - qualification_type
+    - qualification_level
```

Comment:
could we let the data team know if this change please (if you haven't already :D)

Reply:
Yes, I told them last Friday! ✌️ 🚀